paper-api/src/main/java/org/bukkit/event/player/PlayerGameModeChangeEvent.java

```diff
@@ -62,15 +62,15 @@ public Cause getCause() {
     }
 
     /**
-     * <b>Only valid if the cause of the gamemode change was directly due to a command.</b>
+     * <b>Only valid if the gamemode change was caused by a command or the gamemode switcher.</b>
```

Comment:
This is only valid for the gamemode command (not defaultgamemode) since 1.21.9 not sure if it was intended or not but that method was broken anyway before https://github.com/PaperMC/Paper/commit/5613a3597233f66f6f1c496a9d25672c0b51a6bb